### DISCUSSION TITLE
Get all rows in a table via API

### DISCUSSION THREAD

**Discussion by aidenmitchell** [Budibase/budibase#5147] [Help] [answered]
Is there a way to get all of the rows in an internal table via the public API?
Thanks!

**Comment by melohagan** [comment, marked as answer]
Hi @aidenmitchell 

Yes you can, here's the docs: https://docs.budibase.com/reference/post_tables-tableid-rows-search

You'll be able to get the app-id and table-id from the URL in the table view. Then just post an empty query `"query": {}` to return all of the rows.
